Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM python:2-onbuild
+MAINTAINER James Swanick "swanijam@gmail.com"
+
+RUN apt-get update -qq
+RUN apt-get install unzip -y
+RUN apt-get install -y build-essential\
+    git-core
+
+# Initialize a vanilla Harvest project
+RUN harvest init --no-input --no-env project 2>&1 > /dev/null
+
+# Work inside of the generated "project" directory containing the Harvest project.
+WORKDIR project/
+# Add settings and entrypoint script.
+ADD local_settings.py ./project/conf/
+ADD start ./
+ADD global_settings_fix ./
+
+# Default names of the input files.
+ENV METADATA_FILE metadata.csv
+ENV DATA_FILE data.csv
+ADD start /usr/src/app/project/
```

Comment:
Remove this line since it is [redundant](https://github.com/chop-dbhi/redcap-to-harvest/pull/1/files#diff-3254677a7917c6c01f55212f86c57fbfR16).